tests/sentry/tasks/test_daily_summary.py

```diff
@@ -0,0 +1,230 @@
+import copy
+from datetime import datetime, timedelta, timezone
+from unittest import mock
+
+from sentry.constants import DataCategory
+from sentry.issues.grouptype import PerformanceNPlusOneGroupType
+from sentry.models.activity import Activity
+from sentry.models.group import GroupStatus
+from sentry.services.hybrid_cloud.user_option import user_option_service
+from sentry.tasks.summaries.daily_summary import prepare_summary_data, schedule_organizations
+from sentry.tasks.summaries.utils import ONE_DAY
+from sentry.testutils.cases import OutcomesSnubaTest, PerformanceIssueTestCase, SnubaTestCase
+from sentry.testutils.factories import DEFAULT_EVENT_DATA
+from sentry.testutils.helpers.datetime import before_now, freeze_time, iso_format
+from sentry.testutils.helpers.features import with_feature
+from sentry.testutils.silo import region_silo_test
+from sentry.types.activity import ActivityType
+from sentry.types.group import GroupSubStatus
+from sentry.utils.dates import to_timestamp
+from sentry.utils.outcomes import Outcome
+
+
+@region_silo_test
+@freeze_time(before_now(days=2).replace(hour=0, minute=5, second=0, microsecond=0))
+class DailySummaryTest(OutcomesSnubaTest, SnubaTestCase, PerformanceIssueTestCase):
+    def store_event_and_outcomes(
+        self, project_id, timestamp, fingerprint, category, num_times, release=None, resolve=True
+    ):
+        if category == DataCategory.ERROR:
+            data = {
+                "event_id": "a" * 32,
+                "timestamp": iso_format(timestamp),
+                "stacktrace": copy.deepcopy(DEFAULT_EVENT_DATA["stacktrace"]),
+                "fingerprint": [fingerprint],
+            }
+            if release:
+                data["release"] = release
+
+            event = self.store_event(
+                data=data,
+                project_id=project_id,
+            )
```

Comment:
Fyi, i was able to get this to work:
```suggestion
            data = {
                "timestamp": iso_format(timestamp),
                "stacktrace": copy.deepcopy(DEFAULT_EVENT_DATA["stacktrace"]),
                "fingerprint": [fingerprint],
            }
            if release:
                data["release"] = release
            for _ in range(num_times):
                event = self.store_event(
                    data=data,
                    project_id=project_id,
                )
```
but not blocking :)